content/docs/compliance/ato-process.md

```diff
@@ -0,0 +1,70 @@
+---
+menu:
+  docs:
+    parent: compliance
+title: ATO process
+---
+
+This is an outline of a typical ATO process for a cloud.gov customer application. All agencies handle the ATO process in their own way, so you should talk with your agency's security compliance specialists, but this can give you a broad overview.
+
+## What's a FedRAMP Provisional ATO?
+
+[cloud.gov has a FedRAMP Authorization.](https://marketplace.fedramp.gov/#/product/18f-cloudgov?sort=productName) In precise terms, it is a Provisional Authority to Operate (P-ATO) at the Moderate impact level from the FedRAMP Joint Authorization Board (JAB).
+
+It's normal and expected that this is a **"Provisional"** ATO. The JAB does not have the authority to issue an ATO for a system at your agency. Only your own agency has the authority to issue ATOs for systems that your agency uses or operates.
+
+So instead, the JAB issues a *pre-authorization* that your agency can review, including [an audited documentation package]({{< relref "overview/security/fedramp-tracker.md" >}}). If your agency finds that the cloud.gov P-ATO meets their requirements, they can issue an ATO for cloud.gov. This usually looks like this: [template agency ATO letter (.docx)](https://s3.amazonaws.com/sitesusa/wp-content/uploads/sites/482/2017/03/FedRAMP-ATO-Letter-Template-v1.0.docx).
+
+## How customer application ATOs work
+
+### Customer ATO that inherits from cloud.gov ATO (ideal)
+
+Here's what this can look like for the first application to use cloud.gov at an agency. In this diagram, AOs stands for Authorizing Officials -- people who can ATO a system.
+
+{{< diagrams id-prefix="Figure-1." >}}
+graph TD
+CONCEPT(Concept for the first app on cloud.gov at your agency) -->START(1. Discuss with your AOs and get approval to start)
+START -->| | DEV(3. Work on app and start app ATO process)
+START -->| | REVIEW(2. AOs review cloud.gov P-ATO)
+REVIEW -->|AOs approve| CATO(2. AOs issue cloud.gov ATO)
+DEV -->|Can't take next step until AOs issue cloud.gov ATO| APPATO(4. AOs issue app ATO that inherits from cloud.gov ATO)
```

Comment:
I think the fork and numbering for starting work early is unclear. Also we want to avoid giving the impression that ATOs are to be given for individual apps when systems might consist of multiple apps in multiple spaces.

[How about this?](https://mermaidjs.github.io/mermaid-live-editor/#/edit/Z3JhcGggVEQKQ09OQ0VQVChDb25jZXB0IGZvciB0aGUgZmlyc3Qgc3lzdGVtIG9uIGNsb3VkLmdvdiBhdCB5b3VyIGFnZW5jeSkgLS0-IFNUQVJUKDEuIFJlcXVlc3QgYXBwcm92YWwgZnJvbSB5b3VyIEFPcyB0byB1c2UgY2xvdWQuZ292KQpTVEFSVCAtLjIuNS4gaWYgYWxsb3dlZCwgY29tbWVuY2UgZGV2ZWxvcG1lbnQuLT4gREVWClJFVklFVyAtLUFPcyBkaXNhcHByb3ZlLS0-ICBTVE9QKFN0b3AgdXNpbmcgY2xvdWQuZ292KQpTVEFSVCAtLUFPcyByZXF1ZXN0IGNsb3VkIEZlZFJBTVAgcGFja2FnZS0tPiBSRVZJRVcoMi4gQU9zIHJldmlldyBjbG91ZC5nb3YgRmVkUkFNUCBQLUFUTykKUkVWSUVXIC0tQU9zIGFwcHJvdmUtLT4gIENBVE8oMy4gQU9zIGlzc3VlIGNsb3VkLmdvdiBhZ2VuY3kgQVRPKQpDQVRPIC0tPiBERVYoNC4gRGV2ZWxvcCB5b3VyIHN5c3RlbSBhbmQgIGNvbXBsaWFuY2UgPGJyIC8-bWF0ZXJpYWxzIGluaGVyaXRpbmcgZnJvbSBjbG91ZC5nb3YpIApERVYgLS0-UkVRVUVTVEFUTyg1LiBSZXF1ZXN0IEFUTykKUkVRVUVTVEFUTyAtLUFPcyByZXZpZXcgc3lzdGVtIGFuZCBjb21wbGlhbmNlIG1hdGVyaWFscy0tPiBBUFBBVE8oNi4gQU9zIGlzc3VlIHN5c3RlbSBBVE8gKQpBUFBBVE8gLS0-UFJPRChQdXQgeW91ciBzeXN0ZW0gaW4gcHJvZHVjdGlvbikK)

![image](https://user-images.githubusercontent.com/47762/34027828-273708ec-e114-11e7-870e-1ee8c367d9f2.png)

Reply:
Maybe even renumber 2.5 as 1.5 to reflect the ideal, enlightened agency's process!